pom.xml

```diff
@@ -40,9 +103,9 @@
 			<version>3.3.1-SNAPSHOT</version>
 		</dependency>
 		<dependency>
-			<groupId>com.sk89q</groupId>
-			<artifactId>worldedit</artifactId>
-			<version>6.0.0-SNAPSHOT</version>
+			<groupId>com.sk89q.worldedit</groupId>
+			<artifactId>worldedit-bukkit</artifactId>
```

Comment:
worldedit-bukkit is just the new location for the bukkit version of WorldEdit. I can't refer to version 6.0.1 without it.